package.json

```
@@ -63,6 +63,8 @@
     "@angular/router": "14.0.2",
     "@azure/msal-angular": "2.3.3",
     "@azure/msal-browser": "2.27.0",
+    "@microsoft/applicationinsights-angularplugin-js": "^3.0.0",
```

Should we use exact version to be consistent with our other deps?
```suggestion
    "@microsoft/applicationinsights-angularplugin-js": "3.0.0",
```